<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carWindow</class>
 <widget class="QMainWindow" name="carWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>carWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(180, 180, 180)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addCarButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 117, 117);
padding-top: 10px</string>
    </property>
    <property name="text">
     <string>‚ûï</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 117, 117)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="searchCombo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 117, 117);
height: 30px
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 117, 117)</string>
    </property>
    <property name="text">
     <string>üîç</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="carTableWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>521</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 117, 117)</string>
    </property>
   </widget>
   <widget class="QLabel" name="noCarFoundLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(55, 55, 55)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>No Car Found</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCar_List">
    <property name="title">
     <string>Car List</string>
    </property>
   </widget>
   <addaction name="menuCar_List"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
